packages/react-instantsearch/src/components/Highlight.js

```diff
@@ -9,13 +8,4 @@ const Highlight = props => (
   <Highlighter {...props} highlightProperty="_highlightResult" cx={cx} />
 );
 
-Highlight.propTypes = {
```

Comment:
Why is this removed?

Reply:
Don't need to duplicate the props since we don't use it at that stage, we just forward them. All the props are still typed on `Highlighter`.